FirstTask.py

```diff
@@ -0,0 +1,230 @@
+from os import name, system as sys
+
+class BankAccount:
+    def __init__(self, name, password, account_no, balance, account_type):
+        self.name = name
+        self.balance = balance
+        self.account_no = account_no
+        self.account_type = account_type
+        self.password = password
+
+    def account_number(self):
+        return self.account_no
+
+    def pass_code(self):
+        return self.password
+    
+    def check_balance(self):
+        return self.balance
+
+    def check_account_type(self):
+        return self.account_type
+    
+    def deposit_bill(self, bill):
+        if bill >= 0:
+            self.balance += bill
+            return True
+        else:
+            return False
+    
+    def draw_money(self, money):
+        if self.balance >= money:
+            self.balance -= money
+            return True
+        else:
+            return False
+    
+    def account_details(self):
+        print("Account # ", self.account_no)
+        print("Name: ", self.name)
+        print("Balance: ", self.balance)
+        print("Account Type: ", self.account_type)
+  
+class Bank:
+
+    bank_accounts = []
+    logged_in = False
+
+    def total_account(self):
+        return (len(self.bank_accounts))
+
+    def create_account(self, name, code, account_type):
+        self.bank_accounts.append(BankAccount(name, code, 1000+len(self.bank_accounts)+1, 0, account_type))
+
+    def find_account(self, account_number):
+        for i in range(len(Bank.bank_accounts)):
+            if account_number == Bank.bank_accounts[i].account_number():
+                return True, Bank.bank_accounts[i]
+        return False, "Account not found!"
+
+    def login(self, account_number, code):
+        status, response = self.find_account(account_number)
+        if status:
+            if code == response.pass_code():
+                self.logged_in_account = response
+                Bank.logged_in = True
+                return True, response
+        return False, "Account Not Found!"
+
+    def transfer_money(self, sender_account_number, receiver_account_number, amount):
+        status_1, response_1 = self.find_account(sender_account_number)
+        status_2, response_2 = self.find_account(receiver_account_number)
+        if status_1 and status_2:
+            if response_1.draw_money(amount):
+                response_2.deposit_bill(amount)
+                return True
+        else:
+            return False

```

Comment:
Similarly, we should tell which one of the account is not found or both are not found.